src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
+    if (seed != 0)
+    {
+        return new (compiler, CMK_Inlining) RandomPolicy(compiler, isPrejitRoot, seed);
+    }
```

Comment:
What is not clear to me is under what conditions/settings these one of these routines return true?
compInlineRandomStress()
compInlineStress()

My suggestion is to call compStressCompile(stressarea, 50) with a weight of 50% (or 0.5 probability) and select random policy.  If either no stress (jistress=0) or the probability doesn't favor, default is to choose legacy policy.
If random policy gets selected, then it can further use jitstress level as the seed.

Jitstress=0: no stress, always uses legacy policy
jitstress=2: one of random or legacy policy could be used based on method hash.  Further '2' is used as the random seed by random policy.
Jitstress= value other than 0 or 2: Same as above.